.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 {
   "editor.tabSize": 2,
-  "vetur.format.options.useTabs": true,
-  "prettier.enable": false,
-  "prettier.useTabs": true
+  "prettier.enable": true,
+  "prettier.useTabs": true,
+  "files.insertFinalNewline": true,
+  "files.trimFinalNewlines": true
```

Comment:
I use these and am personally a fan, though I can see now wanting the editor to modify the files. If we don't go this route, it might make sense to have eslint rules to enforce the standard instead.

Reply:
Conflicted on this point if it means locking people into specific editors

Reply:
It does not lock anyone to VSCode, just makes anyone using VSCode have a better experience 👍🏼